scopes/lanes/lanes/lanes.main.runtime.ts

```diff
@@ -187,6 +197,110 @@ export class LanesMain {
     return components;
   }
 
+  async getLaneReadmeComponent(name: string): Promise<Component | undefined> {
+    if (!name) return undefined;
+
+    const [lane] = await this.getLanes({ name });
+    const laneReadmeComponent = lane.readmeComponent;
+    if (!laneReadmeComponent) return undefined;
+    const host = this.workspace || this.scope;
+    const laneReadmeComponentId = await host.resolveComponentId(
+      laneReadmeComponent.id.changeVersion(laneReadmeComponent.head)
+    );
+    const readmeComponent = await host.get(laneReadmeComponentId);
+    return readmeComponent;
+  }
+
+  public isLaneReadme(component: Component) {
+    const lanesConfig = component.state.aspects.get(LanesAspect.id)?.config;
+    if (!lanesConfig) return false;
+
+    return Object.keys(lanesConfig).some((lane) => lanesConfig[lane].readme);
+  }
```

Comment:
What does the lanes aspect on a component looks like?

Reply:
yes the component is aware of the fact that is a lane readme since it is a config on the component
It behaves identical to any other meta data on the component, i.e `deprecate`
which is essentially a config on the component

The Lanes config looks like this

```
"config": {
            "teambit.lanes/lanes": {
                "[lane-name]": {
                    "readme": true
                }
            }
        }
```

Reply:
I'm unsure about the config key here, which is the lane-name. 
What if, in the future, you'll want to add a global-config to the lanes aspect? for example, in the future, you'll add a config "defaultSquash" to squash snaps by default upon merge, which is a config that applies for all lanes.

Reply:
I would expect the global config to live outside of the specific lane config, (not under the lane-name) key
something like this, 
```
"config": {
            "teambit.lanes/lanes": {
                config: {
                 // global config that applies to all lanes goes here
                    defaultSquash: true
                 }, 
                "[lane-name]": {
                    "readme": true
                }
            }
        }
```
Or we can get rid of the `config` property and have global lane configs be direct keys of the aspect config

Alternatively,

I could change it to something like this, 
```
"config": {
            "teambit.lanes/lanes": {
                 defaultSquash: true
                "readme": {
                   [lane-name]: true 
                }
            }
        }
```

Reply:
After discussing with @davidfirst ,

Will be going ahead with the alternative approach

```
"config": {
            "teambit.lanes/lanes": {
                 defaultSquash: true
                "readme": {
                   [lane-name]: true 
                }
            }
        }

Reply:
I don't understand why a component should be aware to the `defaultSquash` for example.
even about the readme I'm not entirely sure. I would expect it to have the readme env which is fine. and the lane to point to its read me component. component should know that it's a readme of specific lanes.

Reply:
discussed with @ranm8 to go with the approach of storing lane readme meta data as part of the component config. In future, other lane meta data needs will be implemented on bit.cloud instead of bit